app.py

```diff
@@ -0,0 +1,164 @@
+'''
+Usage:
+    add_person <first_name> <last_name> <role> [--wants_accomodation=N]
+    create_room <room_name> <room_type>
+    reallocate_person <full_name> <new_room_name>
+    load_people <filename>
+    print_people
+    print_allocations [--o=filename]
+    print_unallocated [--o=filename]
+    print_room <room_name>
+    save_state <database>
+    load_state <database>
+    quit
+Options:
+    -h, --help  Show this screen and exit
+    -i --interactive  Interactive Mode
+    --wants_accomodation=<N> [defult: N]
+'''
+
+from app.amity import Amity
```

Comment:
Follow the [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) import recommendation.